app/views/users/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<%# #  <form accept-charset="UTF-8" action="/users" method="post">
+#     <label for="username">Username:</label>
+#     <input id="username" name="user[name]" type="text" />
+      <label for="email">Email:</label>
+#     <input id="email" name="user[email]" type="email" />
+      <label for="password">Password:</label>
+#     <input id="password" name="user[password]" type="password" />
+#     <input name="submit" type="submit" value="Submit" />
+#     <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+   <%# </form> %> %>
+
+<%# <%= form_tag("/users_path", method: "post") do %> %>
+    <%# <%= label_tag(:username, "Create New User:") %> %>
+    <%# <%= text_field_tag(:username) %> %>
+    <%# <%= label_tag(:email, "Email:") %>s %>
+    <%# <%= text_field_tag(:email) %> %>
+    <%# <%= label_tag(:password, "Password:") %> %>
+    <%# <%= text_field_tag(:password) %> %>
+    <%# <%= submit_tag("Submit") %> %>
+  <%# <% end %> %>
+
+<%= form_for @user do |f| %>
+<%= f.label 'username', 'Username:' %>
+  <%= f.text_field :username %><br />
+  <%= f.label 'email', 'Email:' %>
+  <%= f.text_field :email %><br />
+  <%= f.label 'password', 'Password:' %>
+  <%= f.text_field :password %><br />
+  <%= f.submit "Submit" %>
+<% end %>
```

Comment:
Use the appropriate form inputs in here, for example for email use `f.email_field` or `email_field_tag` ...etc depending on that the input is.